src/app/components/chat/window/window.html

```diff
@@ -164,8 +164,12 @@ <h3 ng-if="room.type == Chat_Room.TYPE_PM" class="anim-fade-in-right no-animate-
 							{{ ::msg.loggedOn | amDateFormat:'h:mm A' }}
 						</span>
 
-						<div class="chat-window-message-content-wrap">
-							<div ng-if="::msg.contentRaw.length < 500" class="chat-window-message-content highlight" ng-bind-html="::msg.content"></div>
+						<div ng-if="::msg.contentRaw.length < 500" class="chat-window-message-content-wrap">
+							<div class="chat-window-message-content highlight" ng-bind-html="::msg.content"></div>
+
+							<div class="chat-window-message-mod-tools" ng-if="client.canModerate( room, msg.user )">
+								<span class="jolticon jolticon-friend-remove-2" tooltip="Mute User" ng-click="ctrl.muteUser( msg.user.id )"></span>
```

Comment:
I feel like this should be in an `<a>` tag with ng-click on it. Also put the tooltip on the A tag and change to using `gj-tooltip` instead for this case.